### PR TITLE
Add remix guide

### PR DESCRIPTION
**Pull request by bthaile**
closes https://github.com/onflow/docs/issues/576

Add animated gifs and step by step instructions on how to use Remix on Flow EVM.

### CODE REVIEW

**Review by franklywatson — APPROVED** [review]
Minor tweaks to make things consistent with naming used in other docs

File: docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+1. Create a new file under `scripts`
+2. Paste in above JavaScript code
+3. Click on green play button
+4. Verify the greeting is "Hello World!"
+
+Use the steps below to change the greeting and retrieve the greeting again.
+
+## Updating deployed Smart Contract
+
+<img src="./Remix-update-greeting.gif" width="50%" alt="Update Smart Contract">
```
Comment (gregsantos) [inline docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md:110]: needs closing  tag

File: docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+title: Using Re,ox
```
Comment (gregsantos) [inline docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md:2]:
```suggestion
title: Using Remix
```

Comment (gregsantos) [reply docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md:2]: Might rename directory menu to **Deploying a Smart Contract/**

File: docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+# Using Remix
+
+Remix is an open-source, web-based development environment tailored for EVM smart contract development. It provides developers with a comprehensive suite of tools to write, deploy, and test smart contracts in Solidity. More Information [Remix](https://remix.ethereum.org/)
+
+## Add Flow EVM Network to metamask
+
+<img src="./Remix-adding-metamask-network.gif" width="50%" alt="Add Flow EVM Network" />
+
+Navigate to [Using EVM](../../../using.mdx) page to click on the convenient button to add Flow EVM network information to your metamask.
```
Comment (gregsantos) [inline docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md:17]:
```suggestion
Navigate to [Using EVM](../../../using.mdx) and select the provided button to automatically add Flow EVM network details to your MetaMask wallet.
```

File: docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+getGreeting();
+
+```
+
+1. Create a new file under `scripts`
+2. Paste in above JavaScript code
+3. Click on green play button
+4. Verify the greeting is "Hello World!"
+
+Use the steps below to change the greeting and retrieve the greeting again.
```
Comment (gregsantos) [inline docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-remix.md:106]:
```suggestion
Use the steps below to change the greeting and retrieve the updated greeting.
```

**Review by gregsantos — CHANGES_REQUESTED** [review]

**Review by gregsantos — APPROVED** [review]

**Pull request merged**